### PR TITLE
Delete EUR test and rollout the new page to 100% EUR countries

### PR DESCRIPTION
**Pull request by svillafe**
## What does this change?

The EUR test gave us a 35% uplift so it is our new control.

## Does this affect other platforms - Amp, Apps, etc?
AMP and regular web.
### Banner
picture 657

### Epic
picture 658

### Header
picture 659

### Liveblog Epic
picture 660

### CODE REVIEW

File: common/app/navigation/ReaderRevenueSite.scala [hunk 1]
```diff
@@ -22,7 +22,7 @@ object ReaderRevenueSite extends enumeratum.Enum[ReaderRevenueSite] {
   def isReaderRevenueSiteUrl(url: String): Boolean = getHost(url).exists(hosts.contains)
 
   case object Support extends ReaderRevenueSite {
-    val url: String = s"${Configuration.id.supportUrl}/uk"
+    val url: String = s"${Configuration.id.supportUrl}/"
```
Comment (joelochlann) [inline common/app/navigation/ReaderRevenueSite.scala]: Pretty sure it's fine to ditch the string interpolation and final `/`, as we do with `Configuration.id.contributeUrl` and `Configuration.id.subscribeUrl`

**Comment by svillafe** [comment]
Tested in CODE and is working ✅

**Review by joelochlann — APPROVED** [review]

**Comment by joelochlann** [comment]
💶

**Comment by joelochlann** [comment]
🙌

**Pull request merged**